build.rs

```diff
@@ -0,0 +1,63 @@
+use std::{
+    env,
+    fs::File,
+    io::{self, Write},
+    path::Path,
+    process::Command,
+};
+
+use graphql_client_codegen::{
+    generate_module_token_stream, CodegenMode, GraphQLClientCodegenOptions,
+};
+use syn::Token;
+
+fn main() {
+    let schema_path = format!("{}/schema.docs.graphql", env::var("OUT_DIR").unwrap());
+    let schema_path = Path::new(&schema_path);
+    if !schema_path.exists() {
+        File::create(schema_path).unwrap();
+        let body = reqwest::blocking::get("https://docs.github.com/public/schema.docs.graphql")
```

Comment:
I think that the pre-downloaded and checked in schema file is preferable for reproducible builds.
Would you give me your thoughts?

Reply:
The file is large with about 60000 lines and updated often, I followed the way `octokit/graphql-schema` does:
https://github.com/octokit/graphql-schema

Reply:
> updated often

If so, committing the schema is even so more preferable I think.
Unexpected schema changes may break our builds(eg. type-mismatch), but if the schema is baked, we can diff them and figure out a cause.

I guess `octokit/graphql-schema` is a simple binding library and aims to follow the latest schema, so the codegen is ran in build automatically.

Reply:
I understand. `2c5fa2c` added them